Add BlueMatador Integration Tile (#471)
DataDog/integrations-extras: bluematador/README.md

## Overview

Blue Matador's Datadog integration allows you to send Blue Matador events to the event stream in Datadog.

![eventstream_from_blue_matador][1]

You can use it to enhance your existing dashboards or to correlate with metrics you're collecting in Datadog.

![dashboard][2]

> To see a full list of the events and metrics that Blue Matador monitors that can imported into Datadog, go [here][3].

Reviewer: ```suggestion
For a full list of events and metrics that Blue Matador monitors and that you can import into Datadog, see their [monitors page][3].
```

## Setup

> To get Blue Matador events into Datadog, use a [Datadog API key][4] to create a new notification method in Blue Matador. Note: existing events will not be imported into Datadog, but you will see new events in Datadog as they occur.

Reviewer: ```suggestion
To get Blue Matador events into Datadog, use a [Datadog API key][4] to create a new notification method in Blue Matador. 

**Note**: Already existing events are not imported into Datadog, but new events appear as they occur.
```
